tests/api/api.spec.ts

```diff
@@ -556,6 +556,28 @@ describe("@dataform/api", () => {
           );
         });
     });
+    it("bigquery example with input backticks", () => {
+      return query
+        .compile("select 1 from `tada-analytics.df_integration_test.example_view`", {
+          projectDir: "df/examples/common_v1",
+          projectConfigOverride: { warehouse: "bigquery", defaultDatabase: "tada-analytics" }
+        })
+        .then(compiledQuery => {
+          expect(compiledQuery).equals(
+            "select 1 from `tada-analytics.df_integration_test.example_view`"
+          );
+        });
+    });
+    it("bigquery example with a backslash in regex", () => {
+      return query
+        .compile("select regexp_extract('01a_data_engine', '^(\\d{2}\\w)')", {
+          projectDir: "df/examples/common_v1",
+          projectConfigOverride: { warehouse: "bigquery", defaultDatabase: "tada-analytics" }
+        })
+        .then(compiledQuery => {
+          expect(compiledQuery).equals("select regexp_extract('01a_data_engine', '^(\\d{2}\\w)')");
+        });
+    });
```

Comment:
I'm correct in thinking that the injected code we want in the query should have the same number of backslashes as we input, right? In the lexer they are doubled because it's yet to go through a JS abstraction, whereas here it is going through that abstraction, if you get what I mean...

Reply:
yes, that's right!